project.clj

```diff
@@ -15,13 +15,36 @@
                  [lambdaisland/uri "1.4.54"]
                  [day8.re-frame/http-fx "0.2.2"]
                  [re-pressed "0.3.1"]
-                 [org.metafacture/metafacture-commons "feature-oersi-SNAPSHOT"]
-                 [org.metafacture/metafacture-formeta "feature-oersi-SNAPSHOT"]
-                 [org.metafacture/metafacture-mangling "feature-oersi-SNAPSHOT"]
-                 [org.metafacture/metafacture-runner "feature-oersi-SNAPSHOT"]
-                 [org.metafacture/metafacture-xml "feature-oersi-SNAPSHOT"]
-                 [org.eclipse.xtext/xtext-dev-bom "2.17.0" :extension "pom"]
```

Comment:
I'm not sure if we need to keep this dependency. I remember this was special because it is a transitive dependency of metafacture-fix but is no jar so leiningen had problems to resolve it when not mention it extra (with :extension "pom") and exlcude it in metafacture-fix.

Reply:
Yes, I remember that, but without that, the build was failing, right? Now it seems to work fine without it. The exclusion still seems to be required, it failed without it.

Reply:
This is a little bit strange but it works :ok_woman: